src/styles/blocks/_header.scss

```diff
@@ -34,8 +34,8 @@
       &::before {
         content: "+1 234 555-55-55";
         position: relative;
-        top: 50%;
```

Comment:
Why did it stop working?

Reply:
I think this doesn't work because the element's dimensions are only given by padding(I make position absolute for pseudo before) and this not work for relative elements. I made the pseudo absolute and set the values as in the screenshot
![image](https://user-images.githubusercontent.com/54936044/233770656-927f59c9-74e0-49c5-a18c-52951d4750fe.png)
![image](https://user-images.githubusercontent.com/54936044/233770711-3065476d-93f6-4412-8b0f-8f8fed6aada6.png)